src/QSymbolicsBase/predefined.jl

```diff
@@ -90,6 +90,9 @@ basis(::AbstractSingleQubitGate) = qubit_basis
 basis(::AbstractTwoQubitGate) = qubit_basis⊗qubit_basis
 Base.show(io::IO, x::AbstractSingleQubitOp) = print(io, "$(symbollabel(x))")
 Base.show(io::IO, x::AbstractTwoQubitOp) = print(io, "$(symbollabel(x))")
+Base.:(*)(xs::AbstractSingleQubitGate...) = pauli_simplify(SMulOperator(collect(xs)))
```

Comment:
I would prefer if we avoid complex (potentially expensive, traversing the entire tree deep into the leafs of the arguments) simplification routines in constructors of symbolic expressions. Only extremely basic simplifications should be done automatically without the user requesting them.

An issue like this led to the sympy team having to create a whole new set of constructors without simplification because it made the rest of the codebase too difficult to support otherwise.

Reply:
OK, how about creating a simplify function for the user to manually simplify an expression?

Reply:
Do keep the smaller simplify functions you already have (up to potentially a rename) -- it is nice to have easy-to-access single-purpose simplification routines.

Reply:
Yes, there is a `simplify` exported by Symbolics. `simplify_quantum` sounds OK, but is long. Maybe `qsimplify`?

Reply:
sounds good!